content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,83 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-Use this page to find the type of solution that best fits your needs.
+Depending on your requirements such as ease of maintenance, security, control, available resources, and level of expertise to operate and manage a Kubernetes cluster, there are various solutions available. You can deploy a Kubernetes cluster on a local machine, cloud, on-prem datacenter; or choose a managed Kubernetes cluster. You can also create custom solutions across a wide range of cloud providers, or bare metal environments.
 
-Deciding where to run Kubernetes depends on what resources you have available 
-and how much flexibility you need. You can run Kubernetes almost anywhere, 
-from your laptop to VMs on a cloud provider to a rack of bare metal servers. 
-You can also set up a fully-managed cluster by running a single command or craft 
-your own customized cluster on your bare metal servers.
+Various solutions are available to create a Kubernetes cluster in learning and production environments.
 
 {{% /capture %}}
 
 {{% capture body %}}
 
-## Local-machine Solutions
+## Learning environment
+
+If you want to get familiar with Kubernetes, you can use the Docker-based solutions. You can use tools that are supported by the Kubernetes community or the tools in the ecosystem to set up a Kubernetes cluster on a local machine.
+
+{{< table caption="Local machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
+
+|Community           |Ecosystem     |
+| ------------       | --------     |
+| [Minikube](/docs/setup/minikube/) | [CDK on LXD](https://www.ubuntu.com/kubernetes/docs/install-local) |
+| [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) | [Docker Desktop](https://www.docker.com/products/docker-desktop)|
```

Comment:
wonder if [Kind](https://github.com/kubernetes-sigs/kind)  should be included too ?

Reply:
Kubernetes IN Docker is listed under Community column in https://deploy-preview-14163--kubernetes-io-master-staging.netlify.com/docs/setup/

Reply:
Kind is already listed Line 31: | [Kubernetes IN Docker](https://github.com/kubernetes-sigs/kind) | [Minishift](https://docs.okd.io/latest/minishift/)|